### PR TITLE
Add "super-utilizer?" question

### PR DESCRIPTION
**Pull request by dget**
Waiting on #6, #7.

### CODE REVIEW

**Comment by dget** [comment]
@fureigh !!!!

**Comment by fureigh** [comment]
DAAAAAAAN!

File: functional_tests.py [hunk 1]
```diff
@@ -176,6 +180,38 @@ def test_interested_field_input_works(self):
...
         self.assertIn("field-active", second_field_div.get_attribute('class'))
 
+    def test_superutilizer_field_input_works(self):
+        # Paramedic sees superutilizer field
+        superutilizer_field = self.browser.find_element_by_id('superutilizer-field')
+        self.assertTrue(superutilizer_field.is_displayed())
+
+        # Interested field has an appropriate label
+        superutilizer_label = self.browser.find_element_by_xpath('//fieldset[@id="superutilizer-field"]/legend')
+        self.assertIn(superutilizer_label.text, "Is this person a frequent EMS utilizer?")
```
Comment (fureigh) [inline functional_tests.py:190]: I object to this content, but we can come back to it in the explicitly content-related ticket.

Comment (dget) [inline functional_tests.py:190]: Me too. I agree.

File: functional_tests.py [hunk 2]
```diff
@@ -321,5 +357,33 @@ def test_can_load_page_and_error_on_no_interested(self):
...
+
+        submit_button = self.browser.find_element_by_id('submit')
+        submit_button.click()
+
+        error_box = self.browser.find_element_by_id('error-window')
+        success_message = self.browser.find_element_by_id('success-message')
+
+        self.assertTrue(error_box.is_displayed())
+        self.assertTrue(not success_message.is_displayed())
+        self.assertIn("Please choose if the patient is a frequent EMS utilizer.", error_box.text)
```
Comment (fureigh) [inline functional_tests.py]: 'whether'! :) And I'd say 'select', FWIW.

File: static/js/main.js [hunk 3]
```diff
@@ -59,10 +59,18 @@ $(document).ready(function() {
             return false;
         }
 
+        var superutilizer = getSelectedFieldValue($('#superutilizer-field'));
+        // Show error if empty "superutilizer?" field
+        if (superutilizer === '') {
+            show_error('Please choose if the patient is a frequent EMS utilizer.');
```
Comment (fureigh) [inline static/js/main.js]: 'select whether'

File: templates/index.html [hunk 4]
```diff
@@ -108,6 +108,27 @@
                     </label>
                   </div>
                 </fieldset>
+                <fieldset id="superutilizer-field">
+                  <legend>Is this person a frequent EMS utilizer?</legend>
```
Comment (fureigh) [inline templates/index.html:112]: Flagging for future convo, not blocking PR for it.

File: templates/index.html [hunk 5]
```diff
@@ -108,6 +108,27 @@
...
+                    </label>
+                  </div>
+                  <div class="field">
+                    <label for="superutilizer-no">
+                      No
+                     <input type="radio" name="superutilizer" id="superutilizer-no"></input>
+                    </label>
+                  </div>
+                  <div class="field">
+                    <label for="superutilizer-unsure">
```
Comment (fureigh) [inline templates/index.html:126]: Standardize to -not-sure instead of -unsure?

File: tests.py [hunk 6]
```diff
@@ -62,5 +63,17 @@ def test_response_without_interested_returns_error(self):
...
+        rv_dict = json.loads(rv.data)
+
+        assert rv.status_code == 400
+        assert rv_dict.get("status") == "error"
+        assert "missing_superutilizer" in rv_dict.get("errors")
+
 if __name__ == '__main__':
-    unittest.main()
\ No newline at end of file
+    unittest.main()
```
Comment (fureigh) [inline tests.py:79]: Nbd, but was this intentional?

**Comment by fureigh** [comment]
Back to you, @dget.

File: templates/index.html (hunk 5 above)
Comment (dget) [inline templates/index.html:126]: Leave as unsure, deal with as part of content conversation?

**Pull request merged**